.github/workflows/codeql.yaml

```diff
@@ -19,10 +19,10 @@ jobs:
       with:
         go-version: ${{ env.GO_VERSION }}
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@1ed1437484560351c5be56cf73a48a279d116b78 #v1.1.3
+      uses: github/codeql-action/init@7502d6e991ca767d2db617bfd823a1ed925a0d59 #v1.1.3
```

Comment:
Leaving the version comment in place and unchanged is a bit deceiving.

Reply:
Not sure why dependabot is not updating the comment, but I just deleted the comment.